Resources/Prototypes/_Goobstation/Catalog/Fills/Belt/belts.yml

```
@@ -129,6 +129,42 @@
+      item2:
+        name: jackal
+        whitelist:
+          tags:
+          - WeaponPistolJackal
+  - type: ContainerFill
+    containers:
+      item:
+      - WeaponPistolCasul
+      item2:
+      - WeaponPistolJackal
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)